### ISSUE TITLE
Replay bots upcoming bug

### ISSUE THREAD

**Issue by alerad** [fluffyst/Surftimer#95]
Hey, i'm not sure if you are interested in this, but i've been modding your plugin so that bots are only activated with the replay command.

So.. first thing I stumbled upon was that when I ran a stage bot while the main bot was running, the main bot additional teleports would get messed up, and the replay would bug.

After loosing 6+ hours on fixing it, I found the cause, and It'll probably bring some problems to this plugin at some point:

Since i'm messy I can't pull request it (Already deleted lots of code, added some of mine etc), but I'll explain the bug and fix.
---------------------------------------------------
So... each time a bot gets loaded via replay command,  the variable g_iSelectedReplayBonus and those similar to it get set. Later in the code, at the PlayReplay function, they are used to get the file path to get the additional teleports.

So basically if you have 2 bots running at the same time, this gets bugged and the main bot starts gathering them from another file.

My fix was turning those variables into arrays, kinda like this:

int g_iSelectedReplayType[MAXPLAYERS + 1];
int g_iSelectedReplayBonus[MAXPLAYERS + 1];
int g_iSelectedReplayStage[MAXPLAYERS + 1];
int g_iSelectedReplayStyle[MAXPLAYERS + 1];
int g_iSelectedBonusReplayStyle[MAXPLAYERS + 1];

And setting them according to the bot.

Let me know if you understand any of this, kinda tired and my english might be derping. Gl and thanks for such an awesome open-source plugin.
